src/site/stages/build/process-cms-exports/schemas/input/node-full_width_banner_alert.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+
+module.exports = {
```

Comment:
I noticed you didn't add the `created` field here. Was that left out intentionally?

Reply:
It was my initial one but once the problem is solved I'll go back and add what's missing.